src/app/component/MissionComponent/MissionDetailsCard/MissionDetailsCard.jsx

```diff
@@ -72,27 +74,39 @@ const Row = ({ children, Icon, label }) => {
  */
 const MissionDetailsCard = ({ mission, photoDisabled }) => {
   const snackbarContext = useContext(Snackbar.Context.SnackbarContext);
-  const firebase = getFirebase();
+  const firebase = useFirebase();
   const storage = firebase.storage();
   const classes = useStyles();
   const [file, setFile] = useState(null);
+  const [image, setImage] = useState(null);
 
   if (mission.status === "started") {
     photoDisabled = false;
   } else {
     photoDisabled = true;
   }
 
+  function addImage(img) {
+    const imgURL = Object.values(img)[1];
+    img
+      ? console.log(imgURL) && Mission.update(mission.uid, { deliveryConfirmationImage: imgURL })
+      : snackbarContext.show({
+          message: `Unable to add image to Mission ${mission.uid}: ${error.message}`,
```

Comment:
If you plan on showing an error, this add image should happen in a try-catch so you have an error object to consume